gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceExtractor.java

```diff
@@ -59,7 +37,23 @@
 import com.sforce.async.QueryResultList;
 import com.sforce.soap.partner.PartnerConnection;
 import com.sforce.ws.ConnectorConfig;
-
+import java.io.ByteArrayInputStream;
+import java.io.FileNotFoundException;
+import java.net.URI;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang.StringUtils;
```

Comment:
small nit but Gobblin style guide is enforced here, please add it to your editor: https://gobblin.apache.org/docs/developer-guide/CodingStyle/

Reply:
Done, thanks